mix.exs

```diff
@@ -1,21 +1,13 @@
 defmodule Validatex.MixProject do
   use Mix.Project
 
+  @version "0.3.0"
+
   def project do
     [
       app: :validatex,
-      dialyzer: [
-        plt_add_apps: [:mix],
-        plt_add_deps: :transitive,
-        ignore_warnings: "dialyzer.ignore-warnings",
-        flags: [
-          :unmatched_returns,
-          :error_handling,
-          :race_conditions,
-          :no_opaque
-        ]
-      ],
-      version: "0.3.0",
+      dialyzer: dialyzer_base() |> dialyzer_ptl(System.get_env("SEMAPHORE_CACHE_DIR")),
```

Comment:
Na CI se spousti dializer?

Reply:
Aha, myslel jsem ze zde take udelame semafor

Reply:
jj, udelame, ale nebudem tam poustet dialyzer.

Reply:
aha sry, to jsem nevedel, toz to smaznu